apps/meteor/client/components/message/variants/SystemMessage.tsx

```diff
@@ -78,16 +80,7 @@ const SystemMessage = ({ message, showUserAvatar, ...props }: SystemMessageProps
 			</MessageSystemLeftContainer>
 			<MessageSystemContainer>
 				<MessageSystemBlock>
-					<MessageNameContainer
-						tabIndex={0}
-						role='button'
-						onClick={(e) => user.username && openUserCard(e, user.username)}
```

Comment:
We lost the check `user.username &&` here. I checked the `openUserCard` function and that does not happen there. You sure we can take it out?